service/src/main/resources/HTMLTemplate.html

```diff
@@ -206,20 +206,46 @@ <h3 class="heading heading--two">Orders</h3>
       <dd class="data-list__definition">
         {% if case_data.orders is not empty and case_data.orders.orderType is not empty %}
           {% for type in case_data.orders.orderType %}
-            <span class="u-block">{{ type }}</span>
+            {{ type | fixedlistmapping }}
           {% endfor %}
+          {% if case_data.orders.otherOrder is not empty %}
+            <span class="u-block">{{ case_data.orders.otherOrder }}</span>
+          {% endif %}
+          {% if case_data.orders.emergencyProtectionOrders is not empty %}
+            {% for order in case_data.orders.emergencyProtectionOrders %}
+              {{ order | fixedlistmapping }}
+            {% endfor %}
+            {% if case_data.orders.emergencyProtectionOrderDetails is not empty %}
+              <span class="u-block">{{ case_data.orders.emergencyProtectionOrderDetails }}</span>
+            {% endif %}
+          {% endif %}
         {% else %}
           -
         {% endif %}
       </dd>
     </div>
     <div class="data-list__row">
-      <dt class="data-list__title">Direction and interim orders</dt>
+      <dt class="data-list__title">Direction needed</dt>
       <dd class="data-list__definition">
-        {% if case_data.orders is not empty and case_data.orders.directionsAndInterim is not empty %}
-          {{ case_data.orders.directionsAndInterim }}
-        {% else %}
-          -
+        {% if case_data.orders is not empty %}
+          {% if case_data.orders.orderType is not empty or case_data.orders.directions is not empty %}
```

Comment:
These conditions seems not needed. I suggest removing it.

Reply:
it depends, you could get some unexpected results if you do that, as ccd doesnt reset values on conditional display, so if you chose EPO and then chose the directions that were displayed as a result, then unchecked all orders, the conditional directions would still show based on the orders that no longer exist

Reply:
if thats the kind of functionality we want then sure, but seems wrong to me